docs/dictionary/function/binaryDecode.lcdoc

```diff
@@ -106,13 +106,11 @@ dataTypes as there is data for. Check the <value> that the
 was actually converted. Here is an example:
 
     on convertStuff dataToConvert
-    global headerData,placeholder,bodyData,footerData
-    put binaryDecode("i5x2a24xi2",dataToConvert, \
-
-    headerData,placeholder,bodyData,placeholder,footerData) \
-    into convertResult
-    if convertResult &lt; 3 then return "Error: data was corrupted"
-
+        global headerData,placeholder,bodyData,footerData
```

Comment:
@livecodesam Could you add a line `local convertResult` otherwise this example will not compile in strict compilation mode.